server/src/main/java/io/crate/execution/ddl/views/TransportDropViewAction.java

```diff
@@ -38,8 +38,9 @@
 import org.elasticsearch.threadpool.ThreadPool;
 import org.elasticsearch.transport.TransportService;
 
-import java.io.IOException;
-import java.util.List;
+import io.crate.metadata.RelationName;
+import io.crate.metadata.cluster.DDLClusterStateService;
+import io.crate.metadata.view.ViewsMetadata;
```

Comment:
minor: maybe revert this changes as no real change was made

Reply:
yep, of course, slipped through, thx